### PR TITLE
feat: adding since to S file

### PR DESCRIPTION
**Pull request by CBID2**
### Description of Changes 
This PR turns the "because" entry into "since" and places the revised entry into the S file 
### Related Issue
Closes number 10 in #147

### CODE REVIEW

**Comment by bergerhoffer** [comment]
So, because "since" is already included in the IBM style guide, I think that the correct resolution for number 10 is to just remove the "because" entry. @bredamc Do you agree?

**Comment by mportman12** [comment]
I agree with @bergerhoffer

**Comment by bredamc** [comment]
Yes, I agree -- I added a comment to that effect in the original issue this morning (apologies for putting my comment in the wrong place!) :)

**Comment by CBID2** [comment]
> So, because "since" is already included in the IBM style guide, I think that the correct resolution for number 10 is to just remove the "because" entry. @bredamc Do you agree?

Ok so, I just remove the since entry that I created @bergerhoffer?

**Comment by bergerhoffer** [comment]
> Ok so, I just need to remove the since entry that I created @bergerhoffer?

@CBID2 Yep that's right! Just try to leave line 430 there (the one that says "// EAP: Added "In Red Hat JBoss Enterprise Application Platform,").

Thanks!

**Comment by CBID2** [comment]
> > Ok so, I just need to remove the since entry that I created @bergerhoffer?
> 
> @CBID2 Yep that's right! Just try to leave line 430 there (the one that says "// EAP: Added "In Red Hat JBoss Enterprise Application Platform,").
> 
> Thanks!

Made the change @bergerhoffer! :)

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/s.adoc [hunk 1]
```diff
@@ -428,6 +428,8 @@ Use all lower case, unless used in a title or at the beginning of a sentence.
 *See also*: xref:logical-topology[logical topology], xref:physical-topology[physical topology]
 
 // EAP: Added "In Red Hat JBoss Enterprise Application Platform,"
+
+*See also*:
```
Comment (mportman12) [inline supplementary_style_guide/glossary_terms_conventions/general_conventions/s.adoc:432]: Remove, as there's already a "see also" for this entry.

**Review by mportman12 — CHANGES_REQUESTED** [review]

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/s.adoc (hunk 1 above)
Comment (CBID2) [reply supplementary_style_guide/glossary_terms_conventions/general_conventions/s.adoc]: Done @mportman12

**Review by mportman12 — APPROVED** [review]
LGTM

**Review by IngridT1 — APPROVED** [review]

**Comment by IngridT1** [comment]
LGTM

**Comment by bburt-rh** [comment]
lgtm. Merging

**Pull request merged**